array-manipulation-1/src/tail.js

```diff
@@ -0,0 +1,17 @@
+/* exported tail */
+// pseudocode
+// Create list storage within the function or else it will keep adding on the input by user
+// Identify the first item in the list and know that everything beyond it will be what we want
+// Since we want everything beyond the first item, ignore/skip whatever is in the first position (0)
+// Push everything that is greater than position 0 to the storage in step 1
+// Return the storage
+
+function tail(array) {
+  var tailList = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i > 0) {
```

Comment:
Instead of having this conditional in the loop, you can adjust your initialization to start at `1` rather than `0`, and remove the need for this.

Reply:
Thank you for this sometimes I forget how flexible for loop expressions can be